src/Arrows.js

```diff
@@ -1,68 +1,105 @@
-import React, { Component, PropTypes } from 'react';
+import { default as React, Component, PropTypes } from 'react';
```

Comment:
They're actually the same thing. http://www.2ality.com/2014/09/es6-modules-final.html

Personally, I find it less confusing when I know which is the default export and which are not. I didn't mean for this to make it into the PR.

I'll take it out if you prefer.